lib/tag_editor.dart

```diff
@@ -106,7 +103,7 @@ class _TagsEditorState extends State<TagEditor> {
   var _isFocused = false;
 
   /// Focus node for checking if the [TextField] is focused.
-  FocusNode _focusNode;
+  FocusNode? _focusNode;
```

Comment:
_focusNode is always init in the initState so I don't think it need to be optional. Maybe use the late keyword?
```suggestion
  late FocusNode _focusNode;
```

Reply:
Resolved in https://github.com/panuavakul/material_tag_editor/pull/23/commits/71fca6ed7af9afd5f82a2181cde0b45f8ea33771